onyo/lib/inventory.py

```diff
@@ -201,7 +201,7 @@ def _get_pending_asset_names(self) -> list[str]:
         return names
 
     def _get_pending_dirs(self) -> list[Path]:
-        """Gets inventory dirs that would come into existence due to pending operations.
+        r"""Gets inventory dirs that would come into existence due to pending operations.
```

Comment:
"Gets" -> "Get"

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.